### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by vtleonardo**
## Scope

Refactor the documentation to be more user-friendly. The most important documentation `How to configure and run a node` will not be addressed on this PR since there are some blockers to getting the information required and some clean PRs going on at the same moment. This documentation will be addressed in a subsequent PR. 

For better user experience reviewing this PR, I recommend taking a look at: https://github.com/vtleonardo/alicenet/tree/refactor-docs to see how the docs will appear after they are merged.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Before your submit your cool code, consider the following guidelines:
+
+Search on GitHub for an open or closed PR that relates to your submission. You don't want to duplicate existing efforts.
+
+Be sure that an issue describes the problem you're fixing, or documents the design for the feature you'd like to add. Discussing the design upfront helps to ensure that we're ready to accept your work.
+
+Also, read and follow our [Code of Conduct](CODE_OF_CONDUCT.md) to help us maintain the AliceNet development community a healthy environment.
+
+Finally, check the [Forking and submitting your development work](./docs/CREATING_PR.md) to get your work ready for submission.
```
Comment (diligentis) [inline CONTRIBUTING.md:29]: `docs/` should be move to wiki as we'll likely do a lot of swarming on it there.

File: README.md [hunk 2]
```diff
@@ -1,305 +1,33 @@
...
-make setup
-```
+## Quick Start
 
-To simplify local (or remote via [CodeSpaces](https://github.com/features/codespaces)) development, a
-[devcontainer](https://code.visualstudio.com/docs/remote/containers) is provided. The remainder of the readme should work without additional configuration from within the devcontainer.
+The first step to run an AliceNet blockchain node is to get an AliceNet binary.
 
-## Build AliceNet
+Pre-compiled binaries are available in our [release page](https://github.com/alicenet/alicenet/releases) and instructions on how to build the binary from the source code, can be found in the [Building from Source](./docs/BUILD.md) documentation.
```
Comment (diligentis) [inline README.md:19]: `docs/` should move to wiki.

File: bridge/README.md [hunk 3]
```diff
@@ -1,120 +1,17 @@
 # AliceNet/Bridge
```
Comment (diligentis) [inline bridge/README.md:1]: Just a quick note: I think this is fine for the time being, but we should move `package.json` to the root, at which point we can merge anything needed from this README to the root README as well.

File: bridge/README.md [hunk 4]
```diff
@@ -1,120 +1,17 @@
...
-You can choose between two different commands to run unit tests. The `test-parallel` will be faster but you won't see
-the gas reporter when the tests are finished
-`Note: Gas reporting has been skipped because plugin hardhat-gas-reporter does not support the --parallel flag.)`
-The two commands are respectivetely
+## Requirements
 
-```bash
-$ npm run test
-```
+All the requisites necessary to run smart contracts are covered on the [Building from source](../docs/BUILD.md) documentation.
```
Comment (diligentis) [inline bridge/README.md:9]: `docs/` to wiki.

File: bridge/README.md [hunk 5]
```diff
@@ -1,120 +1,17 @@
...
 
-and
+## Testing
 
-```bash
-$ npm run test-parallel
-```
+Check the [Testing Documentation](../docs/TESTING.md) for more information on how to run the unit tests for the smart contracts.
```
Comment (diligentis) [inline bridge/README.md:13]: `docs/` to wiki.

**Review by diligentis — CHANGES_REQUESTED** [review]
Should move `docs/` to wiki, otherwise looking good!

**Comment by diligentis** [comment]
@vtleonardo I added build/test/running locally to https://github.com/alicenet/alicenet/wiki

**Review by Cr0wn-Gh0ul — CHANGES_REQUESTED** [review]
update it

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+However, before logging an issue be sure to check the current opened and closed issues to see if someone already has submitted the same issue/request.
+
+See the [GitHub help guide](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue) for more information on submit a Github issue.
+
+## Vulnerabilities
+
+Please don't create issues for any security vulnerabilities. Instead, we would appreciate it if you reported them through our vulnerability disclosure form. This allows us to distribute a fix before the vulnerability is exploited.
+
+Additionally, if you believe that you've discovered a security vulnerability, you might qualify for our bug bounty program. Visit our [website](https://www.alice.net/) or our [discord server](https://discord.gg/bkhW2KUWDu) for details.
```
Comment (Cr0wn-Gh0ul) [inline CONTRIBUTING.md:17]: https://discord.gg/8esBWyr8h3

**Review by diligentis — APPROVED** [review]
LGTM!

**Comment by Cr0wn-Gh0ul** [comment]
Follow up with creating configuring node wiki page.

**Pull request merged**